src/Okta.Sdk/IGroupsClient.cs

```diff
@@ -19,5 +19,13 @@ public partial interface IGroupsClient : IAsyncEnumerable<IGroup>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The <see cref="IGroup"/> response.</returns>
         Task<IGroup> CreateGroupAsync(CreateGroupOptions options, CancellationToken cancellationToken = default(CancellationToken));
+
+        /// <summary>
+        /// Removes a specific group rule by id from your organization
+        /// </summary>
+        /// <param name="ruleId">Id of the group rule</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns>A Task that represents the asynchronous operation.</returns>
+        Task DeleteGroupRuleAsync(string ruleId, CancellationToken cancellationToken);
```

Comment:
Should we mark this as obsolete? I have the impression we agreed on that, I might be wrong though.

Reply:
I too remember we were talking about marking it obsolete but then we decided not to do that, that's my impression, but I can't remember why 😕.

Reply:
Do you think it's better to mark it obsolete?